.gitignore

```diff
@@ -1,23 +1,158 @@
+# editor/OS cache files
 .idea/
 .DS_Store
-__pycache__/
 .vscode/
+**/.DS_Store
 
 # jupyter notebooks
 .ipynb_checkpoints/
 
-# packaging
+# Distribution / packaging
+.Python
+build/
+develop-eggs/
 dist/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+wheels/
 *.egg-info/
-build/
+.installed.cfg
+*.egg
+MANIFEST
 
-# Coverage
+# Unit test / coverage reports
+htmlcov/
+.tox/
 .coverage
+.coverage.*
+.cache
+nosetests.xml
 coverage.xml
+*.cover
+.hypothesis/
+.pytest_cache/
+.coverage*
+*.py,cover
+.hypothesis/**
+.pytest_cache/**
 
 # Misc
 results/
 image_files*
 
 # datasets
 cifar*
+
+cleanlab/*.ipynb
+tests/fasttext_data/
+data/
+docs/_build/
+
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
```

Comment:
this is not valid in this repo, please remove this

Reply:
many of these are very common stuff in gitignore, in fact, most of these lines are coming from Github's default Python gitignore. 

I found being proactive much more effective than being retroactive. We will add common things now, but we won't need to add them to gitignore if/when it happens.

Note that prior to this PR, when I set up my environment, I was seeing 30K files to be committed as the `ENV` (default) directory wasn't in gitignore.